app/panel/components/Blocking/trackerButtonRenderHelpers.jsx

```diff
@@ -54,21 +54,42 @@ export const renderKnownTrackerButtons = (
 );
 
 export const renderUnknownTrackerButtons = (
-	handleCliqzTrackerWhitelist, whitelisted, siteRestricted, type
+	handleCliqzTrackerWhitelist, whitelisted, siteRestricted, type, contextType
 ) => {
 	const svgContainerClasses = ClassNames('unknown-svg-container', {
 		whitelisted: whitelisted && !siteRestricted,
 		siteRestricted,
 	});
 
+	let protectedColor;
+	let allowedColor;
+	let restrictedColor;
+	switch (contextType) {
+		case 'palm-theme':
+			protectedColor = '#87f0fb';
+			allowedColor = '#b8e986';
+			restrictedColor = '#ff7e74';
+			break;
+		case 'leaf-theme':
+			protectedColor = '#2e3b80';
+			allowedColor = '#326c45';
+			restrictedColor = '#963939';
+			break;
+		default:
+			protectedColor = '#00AEF0';
+			allowedColor = '#96c761';
+			restrictedColor = '#00AEF0';
+			break;
+	}
+
```

Comment:
I like inlining the SVGs, as per my other comment, but would it be better to set these attributes through CSS, so that there is a CSS file that is the single source of truth for all the styling values for any given theme? This does have the advantage of being a bit simpler and more direct than the CSS way, so I'm not sure. @Eden12345 - thoughts?

Reply:
I think the CSS files should be the single source of truth, as you're saying. Since we're already using inline SVGs, it won't be too difficult to move these colors to a CSS file.

Reply:
The CSS is now the single source of truth!